### DISCUSSION TITLE
New_Programs (Incremental Data)

### DISCUSSION THREAD

**Discussion by gprao08** [Sunbird-cQube/community#866] [General]
@RNarmatha 
@bommala
As we discussed earlier we have upgrade our version from 5.0.3 to 5.0.5, Now we ingest June 2023 month Teacher's Attendance data, Can we ingest every month data without erasing old data because we face same issue in the previous version and that is the reason we upgrade to version 5.0.5. 
If so same procedure for New_programs?

**Comment by bommalla** [comment]
@gprao08 
hope this message finds you well. I am writing to inform you of an important update regarding the release of version v5.0.5.

In this latest version, v5.0.5, a new feature has been implemented that allows for the ingestion of monthly data without the risk of losing existing data. The procedure for ingesting monthly data remains unchanged from previous versions.

Furthermore, for new programs, it is no longer necessary to schedule the "Run_adopter" processor group. Instead, please schedule only the "data_moving_local" and "ingest_data" processor groups. It's crucial to note that scheduling the "Run_adopter" processor group may result in the loss of previous data.

**Reply by gprao08** [reply]
@bommalla 
Thanks for your reply. Could you plz correct the below steps for "data_moving_local" and "ingest_data" processor groups.
{   
	"processor_group_name": "data_moving_local",
	"scheduled_at": "0 20 15 * * ?",
	"program_name": "sch_att"
}

{   
	"processor_group_name": "ingest_data",
	"scheduled_at": "0 20 15 * * ?",
	"program_name": "sch_att"
}

**Reply by bommalla** [reply]
@gprao08 

For "data_moving_local" processor group program name is not required.Please find the below request body.

{
"processor_group_name": "data_moving_local",
"scheduled_at": "0 20 15 * * ?"
}
{
"processor_group_name": "ingest_data",
"scheduled_at": "0 20 15 * * ?",
"program_name": "sch_att"
}

**Reply by gprao08** [reply]
@bommalla 
Noted. Thanks for your reply.
